src/commonmark/en/content/developer/web-api/new-tracker.md

```diff
@@ -0,0 +1,2085 @@
+
+`Tracked Entities` are the root object for the Tracker model.
+
+| Property | Description | Required | Immutable | Type | Example |
+|---|---|---|---|---|---|
+| trackedEntity | The identifier of the tracked entity. Generated if not supplied | No | Yes | String:Uid | ABCDEF12345 |
+| trackedEntityType | The type of tracked entity. | Yes | No | String:Uid | ABCDEF12345 |
+| createdAt | Timestamp when the user created the tracked entity. Set on the server. | No | Yes | Date:ISO 8601 | YYYY-MM-DDThh:mm:ss |
+| createdAtClient | Timestamp when the user created the tracked entity on the client. | No | No | Date:ISO 8601 | YYYY-MM-DDThh:mm:ss |
+| updatedAt | Timestamp when the object was last updated. Set on the server. | No | Yes | Date:ISO 8601 | YYYY-MM-DDThh:mm:ss |
+| updatedAtClient | Timestamp when the object was last updated on the client. | No | No | Date:ISO 8601 | YYYY-MM-DDThh:mm:ss |
+| orgUnit | The organisation unit where the user created the tracked entity. | Yes | No | String:Uid | ABCDEF12345 |
+| inactive | Indicates whether the tracked entity is inactive or not. | No | No | Boolean | Default: False, True |
+| deleted | Indicates whether the tracked entity has been deleted. It can only change when deleting. | No | Yes | Boolean | False until deleted |
+| geometry | A  geographical representation of the tracked entity. Based on the "featureType" of the TrackedEntityType. | No | No | GeoJson | {<br>"type": "POINT",<br>"coordinates": [123.0, 123.0]<br>} |
+| storedBy | Client reference for who stored/created the tracked entity. | No | No | String:Any | John Doe |
+| attributes | A list of tracked entity attribute values owned by the tracked entity. | No | No | List of TrackedEntityAttributeValue | See Attribute |
+| enrollments | A list of enrollments owned by the tracked entity. | No | No | List of Enrollment | See Enrollment |
+| relationships | A list of relationships connected to the tracked entity. | No | No | List of Relationship | See Relationship |
+| programOwners | A list of organisation units that have access through specific programs to this tracked entity. See "Program Ownership" for more. | No | No | List of ProgramOwner | See section "Program Ownership" |
+
+> ***Note***
+> `Tracked Entities` "owns" all `Tracked Entity Attribute Values` (Or "attributes" as described in the previous table). However, `Tracked Entity Attributes` are either connected to a `Tracked Entity` through its `Tracked Entity Type` or a `Program`. We often refer to this separation as `Tracked Entity Type Attrbiutes` and `Tracked Entity Program Attributes`. The importance of this separation is related to access control and limiting what information the user can see.
+>
+> The "attributes" referred to in the `Tracked Entity` are `Tracked Entity Type Attributes`.
+
+
+### Enrollment
+`Tracked Entities` can enroll into `Programs` for which they are eligible. We represent the enrollment with the `Enrollment` object, which we describe in this section.
```

Comment:
``Tracked Entities` can enroll into `Programs` for which they are eligible` - I think this opens up for questions. What do we mean by eligible? Can we point to another section in the docs?

Reply:
clarified.